.github/workflows/test-release.yml

```diff
@@ -0,0 +1,66 @@
+name: Publish Python 🐍 distribution 📦 to PyPI and TestPyPI
+
+on: push
+
+jobs:
+  build:
+    name: Build distribution 📦
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        persist-credentials: false
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.x"
+    
+    - name: Install Node
+      uses: actions/setup-node@v2
+      with:
+        node-version: '18'
+
+    - name: Install npm dependencies
+      run: |
+        cd js
+        npm ci --verbose
+        npm run build --verbose
+
+    - name: Install pypa/build
+      run: >-
+        python3 -m
+        pip install
```

Comment:
Is it worth using `uv` here? no external dependencies here so not a lot to reconcile but nonetheless.

Reply:
Yeah I don't know that it is worth it - we'd just be installing one package